mcp-server/README.md

```diff
@@ -0,0 +1,57 @@
+# BanyanDB MCP Server
```

Comment:
Please move the docs into here https://github.com/apache/skywalking-banyandb/blob/main/docs and add a menu into the https://github.com/apache/skywalking-banyandb/blob/main/docs/menu.yml file.